README.md

```diff
@@ -31,6 +31,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.3</b> or higher
+ * curl [production install] or php7.3-curl [development install]
```

Comment:
@johnsaigle do you know if `curl` is necessary ? I can only see the need for php-curl from the linked PR

Reply:
I think `curl` is required for prod and dev but I can't point to why.

`php-curl` is required only for development installs.